.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+### Summary of Issue
+
+<A brief summary of the issue (used for release notes).>
+
+### Issue Type
+
+- [ ] Bug
+- [ ] Enhancement (POEM: N/A or POEM number at https://github.com/OpenMDAO/POEMs)
```

Comment:
Is it intended that the POEM number be in the square brackets or in the parens?

Would one of the following be more clear?  I think I prefer the latter:

```
[ ] Enhancement, POEM #[   ]  (N/A or POEM number at https://github.com/OpenMDAO/POEMs)
```
```
[ ] Enhancement
    POEM #[   ]  (N/A or POEM number at https://github.com/OpenMDAO/POEMs)
```

Reply:
Agreed, I'll make that change.